owl/SOMA-OBJ.owl

```diff
@@ -740,39 +755,39 @@ InverseObjectProperties(SOMA:isDepositOf SOMA:isOntopOf)
 ObjectPropertyDomain(SOMA:isDepositOf <http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#PhysicalObject>)
 ObjectPropertyRange(SOMA:isDepositOf <http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#PhysicalObject>)
 
-# Object Property: <http://www.ease-crc.org/ont/SOMA.owl#isDesignOf> (is design of)
```

Comment:
I suppose this is related to whatever happened to hasDesign.

Reply:
`is design of` happend to be a duplicate of `is design for`.